downstream/modules/platform/ref-hub-variables.adoc

```diff
@@ -0,0 +1,163 @@
+[id="ref-hub-variables"]
+
+= {HubNameMain} variables
+
+[cols="50%,50%",options="header"]
+|====
+| *Variable* | *Description* 
+| *`automationhub_admin_password`* | Required
+| *`automationhub_api_token`* a| If upgrading from {PlatformNameShort} 2.0 or earlier, you must either:
+
+* provide an existing {HubNameMain} token as `automationhub_api_token`, or 
+
+* set `generate_automationhub_token` to `true` to generate a new token
+
+Generating a new token invalidates the existing token.
+| *`automationhub_authentication_backend`* a| This variable is not set by default. 
+Set it to `ldap`` to use LDAP authentication. 
```

Comment:
Is this a typo: `ldap`` ?

In line 19, it appears as `'ldap'`

Reply:
No. Text variables should be in quotation marks, so `'ldap'`

Reply:
Sorry, I wasn't clear in my comment. The text variable only shows one quotation mark when rendered.